### ISSUE TITLE
NVDA does not reports, when text is unselected in browse mode

### ISSUE THREAD

**Issue by kvark128** [nvaccess/nvda#10731]
### Steps to reproduce:
1. Select some text in browse mode. Can use any document with browse mode.
2. Press any arrow key to cancel the selection.
### Actual behavior:
NVDA reports the character under the virtual cursor, but does not report that the text was unselected.
### Expected behavior:
NVDA should announce that the text was unselected.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2019.3beta3
#### Windows version:
Windows 7 and Windows 10
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by feerrenrut** [comment]
Which browser are you using? Can you please try NVDA 2019.2.1 and report the behavior?

**Comment by ehollig** [comment]
I can produce this using alpha-19538,b21a299b and Chrome. This does not occur with NVDA 2019.2.1.
1. Select a line of text using shift+end
2. Press left arrow to move to the previous letter.
Actual: NVDA announces the letter.
Expected: NVDA announces text unselected followed by the letter.

**Comment by kvark128** [comment]
The problem occurs in any application where there is browse mode (Firefox, chrome, Thunderbird etc.). In NVDA 2019.2.1 everything works correctly.
